x/instrumentation/test_instrumentation.go

```diff
@@ -44,9 +47,9 @@ const (
 )
 
 func PeggyCheckpoint(logger log.Logger, kind string, keysAndValues ...interface{}) {
-	logger.Debug(peggyTestMarker, append([]interface{}{kindMarker, kind}, keysAndValues...)...)
+	logger.Debug(peggyTestMarker, append([]interface{}{kindMarker, kind, "cmdline", strings.Join(os.Args, " ")}, keysAndValues...)...)
```

Comment:
Is this just logging the command line arguments out as a debug line?

Reply:
Yes